README.md

```diff
@@ -1,13 +1,74 @@
-# Taskmaster [![42](https://i.imgur.com/9NXfcit.jpg)](i.imgur.com/9NXfcit.jpg)
 
-Taskmaster is a 42 project. It is an implementation of a **process supervisor** and aims at getting familiar with job control mechanisms on UNIX-like operating systems.
+<br />
+<p align="center">
+  <a href="https://github.com/pscott/TaskMaster">
+    <img src="assets/logo.png" alt="Logo" width="90" height="90">
```

Comment:
Curious where this logo comes from? :)

Reply:
I made the logo from 2 black and white no-copyright logos. Do you like it ? The old man for "master" and the screwdriver for the dirty tool/task/job aspect. ^^

Reply:
Yup great logo!